### PR TITLE
Fcc branch

### PR DESCRIPTION
**Pull request by fauziyahzeeyah**
- [ ]  Add linters
- [ ]  Update Readme. md
- [ ]  Add gitignore

### CODE REVIEW

**Review by Debas-31 — COMMENTED** [review]
Hi @fauziyahzeeyah  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :negative_squared_cross_mark:  :negative_squared_cross_mark:  :negative_squared_cross_mark: 

- [ ] You have submitted a project PR without a review of your past trial review. Please review it your code at least to have two inline comments in the PR. Please refer [this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) how to review a code 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by fauziyahzeeyah — COMMENTED** [review]
### Added

-  Readme.md files
-  Linters
-  gitignore
   
    **Request review.**

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @fauziyahzeeyah, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Aspects to improve ♻️

- [ ] Check the comments under the review.
- [ ] Unfortunately, your review does not mention any suggestions that the person should change in his/her code. It is a project requirement that you add at least 2 comments in the review pointing to suggestions for the 'student'. Remember to use the inline comments feature and to be polite and explain 'why' you are giving the suggestions. Kindly perform a new review on this project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (fauziyahzeeyah) [inline .gitignore:1]: -  I think adding a test. md to gitignore would be perfect.

File: README.md [hunk 2]
```diff
@@ -1 +1,86 @@
-# MF
\ No newline at end of file
+<a name="readme-top"></a>
```
Comment (fauziyahzeeyah) [inline README.md:1]: -  Your title should be descriptive as well ☺

File: README.md [hunk 3]
```diff
@@ -1 +1,86 @@
...
+If you like this project please give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[Question_1]**
+ Why did I add MIT.md?
+  - [Answer_1]
+   I added MIT license because it gives users express permission to reuse code for any purpose.
```
Comment (fauziyahzeeyah) [inline README.md:73]: -  There is no MIT license in your code. Add MIT license to your code or you change question 1 👀

**Review by fauziyahzeeyah — COMMENTED** [review]
### Changes Required ❎ ❎ 

gif


Hi @fauziyahzeeyah 
     You are almost there, just some little changes left and you can move to the next project.
 
- [ ]  Check the comments under the review

Cheers and Happy coding!👏👏👏

### Optional Suggestions

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


   _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @fauziyahzeeyah 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Polite and professional comments. ✔️ 
 - Good use of inline comments. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**